test/fixtures/get-test-server.ts

```diff
@@ -1,54 +1,46 @@
 import type { ExecutionContext } from "ava"
-import type { Axios } from "axios"
-import type { NextApiRequest } from "next"
+import defaultAxios from "axios"
+import type { Middleware } from "edgespec"
+import { getTestServer as getTestEdgeSpecServer } from "edgespec/testing/ava"
 import type { TypedAxios } from "typed-axios-instance"
 
 import type { Database, Routes } from "@seamapi/fake-template"
 
-import getServerFixture from "nsm/get-server-fixture.ts"
-import type { NextApiHandler, NextApiResponse } from "nsm/types/nextjs.ts"
-
-import { type DatabaseFixture, getTestDatabase } from "./get-test-database.ts"
-
-export type { SimpleAxiosError } from "nsm/get-server-fixture.ts"
-
-type GetServerFixture = typeof getServerFixture.default
-
-type ServerFixture<TSeed = true> = DatabaseFixture<TSeed> &
-  Omit<Awaited<ReturnType<GetServerFixture>>, "axios"> & {
-    axios: TypedAxios<Routes>
-    get: Axios["get"]
-  }
-
-interface ApiRequest extends NextApiRequest {
-  db?: Database | undefined
-}
+import { getTestDatabase } from "./get-test-database.ts"
 
 export const getTestServer = async <TSeed extends boolean>(
   t: ExecutionContext,
   { seed: willSeed }: { seed?: TSeed } = {},
-): Promise<ServerFixture<TSeed>> => {
+) => {
   const { db, seed } = await getTestDatabase(t, {
     seed: willSeed ?? true,
   })
 
-  const fixture = await (getServerFixture as unknown as GetServerFixture)(t, {
-    middlewares: [
-      (next: NextApiHandler) => (req: ApiRequest, res: NextApiResponse) => {
-        req.db = db
-        return next(req, res)
-      },
-    ],
+  const withDb: Middleware<unknown, { db: Database }> = async (
+    req,
+    ctx,
+    next,
+  ) => {
+    ctx.db = db
+    return await next(req, ctx)
+  }
+
+  const { port } = await getTestEdgeSpecServer(t, {
+    middleware: [withDb],
   })
 
-  fixture.axios.defaults.headers.common["authorization"] =
-    `Bearer ${seed.api_key}`
+  const serverURL = `http://localhost:${port}`
+
+  const axios = defaultAxios.create({
+    baseURL: serverURL,
+  }) as TypedAxios<Routes>
+
+  axios.defaults.headers.common["authorization"] = `Bearer ${seed.api_key}`
 
   return {
-    ...fixture,
-    // @ts-expect-error Current version of axios has upstream type issue.
-    get: fixture.axios.get.bind(fixture.axios),
+    serverURL,
+    axios,
```

Comment:
Should we move to this now?
```suggestion
    client,
```

Reply:
yep works for me